.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       CI: true
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x]
```

Comment:
I update ava to the latest version to have better-debugging support and they drop support for node 8.